docs/_snippets/flutter_todos_tutorial/lib/app.dart.md

```diff
@@ -0,0 +1,38 @@
+```dart
+import 'package:flutter/material.dart';
```

Comment:
Where possible, I would highly recommend/prefer to avoid having snippets which are identical to the actual implementation. In those cases, we can just reference the actual files in GitHub to avoid duplication and to ensure the snippets are always in sync and up to date. Let me know if that makes sense or if you have any questions, thanks!

Reply:
100% Agree, with comment.

- Linking to the tutorial on `master` syncs to master, but the tutorial text might reflect an older version.
- Making a snippet makes sure the website has a current snapshot at the time of writing. I've seen this happen for meteor and nextJS docs after major refactors.
  -  **What do you think about linking to a specific commit?** (then, on any tutorial update, one can find/replace to point to the new commit URL. So one would be deliberate in updating the version of linked code files.)
- I was following the use of snippers of what I saw in other tutorials. I suggested an edit to `Contributing` in the wiki. See #3056. I'm not wed to the language.

Recommend not letting this be blocking. This can be a later re-factor. It's probably about 1 hour of work.

Reply:
I think we can do a merge of the tutorial’s code branch and then reference the master branch with the new files, so @chonghorizons doesn’t need to change the files location when we merge this PR, in case there are a lot of entire files being used as snippets, let me know what you both think 👍

Reply:
otherwise we can simplify the snippets with the chunk of code we’re referencing instead of pasting the whole file